contrib/debian/changelog

```diff
@@ -179,7 +241,7 @@ bitcoin (0.5.3-natty0) natty; urgency=low
 bitcoin (0.5.2-natty1) natty; urgency=low
 
   * Remove mentions on anonymity in package descriptions and manpage.
-    These should never have been there, bitcoin isn't anonymous without
+    These should never have been there, bitcoin isnt anonymous without
```

Comment:
changes starting from this line look more like a rollback, bringing errors/misspellings...